src/Psalm/Internal/Scanner/DocblockParser.php

```diff
@@ -101,8 +101,10 @@ public static function parse(string $docblock, int $offsetStart): ParsedDocblock
                 unset($lines[$k]);
             } else {
                 // Strip the leading *, if present.
-                $lines[$k] = str_replace("\t", ' ', $line);
-                $lines[$k] = preg_replace('/^ *\*/', '', $line);
```

Comment:
It looks like a bug actually, the tab expansion was futile.

Reply:
Yeah, this was definitely a bug. Forgot to mention it in the PR. I fixed it the way I think it was intended, but I'm not sure what it was designed to do in what case though...